cppForSwig/Signer.h

```diff
@@ -209,12 +209,23 @@ class ScriptSpender
 /////////////////// Spender that doesn't require resolution ///////////////////
 class ScriptSpender_Signed : public ScriptSpender
 {
-public:
-   ScriptSpender_Signed(const UTXO& utxo)  : ScriptSpender(utxo)
+protected:
+   ScriptSpender_Signed(const UTXO& utxo) : ScriptSpender(utxo)
    { }
    ~ScriptSpender_Signed() override = default;
+public:
+   virtual void setSignedWitnessData(const BinaryData &inputSig, const int itemCount) {}
+   virtual void setSignedScript(const BinaryData& inputSig) {}
```

Comment:
You'd better make them abstract, or throw an "unimplemented" exception in each of them.

Reply:
Done